manifests/config.pp

```diff
@@ -11,6 +11,8 @@
   Hash $apmserver_config_custom                   = $apmserver::apmserver_config_custom,
   String $ori_ext                                 = $apmserver::_ori_ext,
   Optional[String] $apmserver_default_config_file = $apmserver::apmserver_default_config_file,
+  String $config_owner                            = $apmserver::config_owner,
+  String $config_group                            = $apmserver::config_group,
 ){
```

Comment:
I'd rather not force a default to 'root'.   This would be more to my liking...
Optional[String] $config_owner                            = undef,
Optional[String] $config_group                            = undef,